mobile/function/poi_info_page_functions.js

```diff
@@ -22,6 +21,7 @@ export const showPoiInfo = (poiId) => {
         const json = await response.json();
 
         setData(json);
+        console.log(json);
```

Comment:
ook console.log weg halen tenzij je die gebruikt die er met een reden is natuurlijk